render/filters.go

```diff
@@ -58,12 +58,26 @@ func ColorConnected(r Renderer) Renderer {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc func(report.Node) bool
+	FilterFunc     func(report.Node) bool
+	ReportFiltered bool // false means we don't report stats for how many are filtered
```

Comment:
Also, do we *ever* want the stats to include pseudo nodes filtered? i.e. why not just check if the node is a pseudo before incrementing the counter?